specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -5331,6 +5450,135 @@
         }
       }
     },
+    "IpAddressAggregate": {
+      "description": "The key value pair for properties.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID for the entree",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "The tenant ID",
+          "type": "string"
+        },
+        "serviceId": {
+          "description": "The service ID",
+          "type": "string"
+        },
+        "ipAddress": {
+          "description": "The Bad Ip Address",
```

Comment:
nit. casing. Bad -> bad. Also please bear in mind that the description here WILL end up in both documentation AND intellisense prompt. So please be as thorough and informative as possible. "bad" is too generic here. For example: "The list of IP addresses to does not conform to blah blah."

Reply:
Fixed